Simbody/src/SimbodyMatterSubsystemRep.cpp

```diff
@@ -5400,6 +5401,55 @@ void SimbodyMatterSubsystemRep::multiplyByMInv(const State& s,
 }
 //............................. CALC M INVERSE F ...............................
 
+//==============================================================================
+//                            MULTIPLY BY SQRT M INV
+//==============================================================================
+// Calculate u = sqrt(M^-1) v. We also get spatial accelerations A_GB for
+// each body as a side effect.
+// This Subsystem must already be realized through Dynamics stage.
```

Comment:
OK. I am now up to date with the changes so I modified stage realization requirement accordingly MultiplyBySqrtMInv is now computing spatial velocities as a side-effect not spatial accelerations.